hw9.py

```diff
@@ -0,0 +1,11 @@
+### generator
+num = int(input('Input number: '))
+
+def Fibonacci(n):

```

Comment:
1. Function names in Python should be in lowercase: `def fibonacci(n):`
2. `n` variable is unused here. You use `num` instead, but `num` is a variable from the outer scope, and in this case, your function shouldn't work with variables from the outer scope.

Reply:
done